libs/api/app/data-access/src/lib/entity/app-transaction.entity.ts

```diff
@@ -22,7 +22,7 @@ export class AppTransaction {
   destination?: string
   @ApiProperty()
   @Field(() => [String], { nullable: true })
-  errors?: string[]
+  errors?: any
```

Comment:
In order for this to work, the `@Field()` decorator would need to be updated and after that, the `admin-sdk` need to be re-generated and the Admin UI needs to be accommodated to display the content of the 'JSON' type. 

I think for now we should stick with `string[]` and add the error as a string to the array, and later find a way how to store these errors on our AppTransaction.

Reply:
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/36491/170145262-663aa29c-e6bc-414e-a097-ad9d84eb9775.png">